linux_os/guide/system/accounts/accounts-pam/set_password_hashing_algorithm/set_password_hashing_algorithm_logindefs/rule.yml

```
@@ -50,10 +50,22 @@ references:
     stigid@sle15: SLES-15-010260
     stigid@ubuntu2004: UBTU-20-010404
 
-ocil_clause: 'it does not'
+ocil_clause: '"ENCRYPT_METHOD" is not set to SHA512'
```

```suggestion
ocil_clause: 'ENCRYPT_METHOD is not set to SHA512'
```
In `set_password_hashing_algorithm_libuserconf` the config key is not in quotes.